pkg/asset/imagebased/image/image.go

```diff
@@ -60,8 +60,8 @@ func (i *Image) Generate(_ context.Context, dependencies asset.Parents) error {
 
 // PersistToFile writes the ISO image in the assets folder.
 func (i *Image) PersistToFile(directory string) error {
-	if i.Config == nil {
-		return fmt.Errorf("imagebased installation configuration is nil")
+	if i.Config == nil || len(i.IgnitionByte) == 0 {
```

Comment:
When will len(i.IgnitionByte) == 0 ?

Reply:
As far as I can understand, if [this](https://github.com/openshift/installer/blob/master/pkg/asset/imagebased/image/ignition.go#L65) errors, then [this](https://github.com/openshift/installer/blob/master/pkg/asset/imagebased/image/image.go#L49) will end up with an empty `IgnitionByte` (or perhaps this `Generate` will not even be called, which still leaves the `IgnitionByte` empty). Although both `Generate` methods will fail, the `PersistToFile` will be called. Does this make sense?